### PR TITLE
POWER9 platforms switch to GCC6

### PR DESCRIPTION
**Pull request by stewart-ibm**
Fixes: https://github.com/open-power/op-build/issues/755
Signed-off-by: Stewart Smith <stewart@linux.vnet.ibm.com>

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/898)

### CODE REVIEW

File: openpower/package/hostboot/p9Patches/hostboot-0008-revert-ipmi-progress-sensor-sync.patch [hunk 1]
```diff
@@ -0,0 +1,138 @@
+From 9375e99c7a748cba00994b582be7571b8cc271dc Mon Sep 17 00:00:00 2001
+From: William G. Hoffa <wghoffa@us.ibm.com>
+Date: Thu, 23 Feb 2017 21:33:34 -0500
+Subject: [PATCH] Revert "Sending Firmware progress sensor is now synchronous"
+
+This reverts commit 9b188cdec934275ce65eab3ade91398fdec4ddb9.
```
Comment (dcrowell77) [inline openpower/package/hostboot/p9Patches/hostboot-0008-revert-ipmi-progress-sensor-sync.patch]: Is this really a GCC6 change?  It seems unrelated.

**Comment by stewart-ibm** [comment]
Dan <notifications@github.com> writes:
>> @@ -0,0 +1,138 @@
> +From 9375e99c7a748cba00994b582be7571b8cc271dc Mon Sep 17 00:00:00 2001
> +From: William G. Hoffa <wghoffa@us.ibm.com>
> +Date: Thu, 23 Feb 2017 21:33:34 -0500
> +Subject: [PATCH] Revert "Sending Firmware progress sensor is now synchronous"
> +
> +This reverts commit 9b188cdec934275ce65eab3ade91398fdec4ddb9.
>
> Is this really a GCC6 change?  It seems unrelated.

There was a warning/error thrown in the code that was reverted, so by
doing the revert (which needed to be done anyway, as it was missed in p9
hostboot but done in p8 hostboot), we fix the problem by removing the
problematic code.

-- 
Stewart Smith
OPAL Architect, IBM.
